src/components/DuosHeader.jsx

```diff
@@ -365,7 +365,7 @@ const DuosHeader = (props) => {
   };
 
   const makeNotifications = () => {
-    return state.notificationData.map((d, index) => <Notification notificationData={d} key={index} />);
+    return state.notificationData.map((d, index) => <Notification notificationData={d} key={index} divKey={index} />);
```

Comment:
```suggestion
    return state.notificationData.map((d, index) => <Notification notificationData={d} key={index} index={index} />);
```

I like Greg's approach in the other PR.

Reply:
Ah, I just saw this after merging :/. Will update